pygmt/src/coast.py

```diff
@@ -183,10 +188,22 @@ def coast(self, **kwargs):
     {V}
     """
     kwargs = self._preprocess(**kwargs)  # pylint: disable=protected-access
+
+    if "clip" in kwargs and kwargs["clip"][0] in "land":
+        kwargs["G"] = True
+        kwargs.pop("clip")
+    elif "clip" in kwargs and kwargs["clip"][0] in "water":
+        kwargs["S"] = True
+        kwargs.pop("clip")
+    elif "clip" in kwargs and kwargs["clip"][0] in "end":
+        kwargs["Q"] = True
+        kwargs.pop("clip")
```

Comment:
Could you explain what `kwargs["clip"][0] in "land"` does? I don't understand why you need to take the first value (`[0]`), and why the check is using `in` rather than `==`.

Reply:
Not sure what I did there, thanks @weiji14 😉.